Sources/TuistDependencies/SwiftPackageManager/Models/WorkspaceState.swift

```diff
@@ -0,0 +1,28 @@
+/// The Swift Package Manager workspace information.
+/// It decodes data encoded from WorkspaceState.swift: https://github.com/apple/swift-package-manager/blob/ce50cb0de101c2d9a5742aaf70efc7c21e8f249b/Sources/Workspace/WorkspaceState.swift
+/// In particular, we are interested in the ManagedDependency.swift: https://github.com/apple/swift-package-manager/blob/ce50cb0de101c2d9a5742aaf70efc7c21e8f249b/Sources/Workspace/ManagedDependency.swift
+/// Fields not needed by tuist are commented out and not decoded at all.
+struct WorkspaceState: Decodable, Equatable {
```

Comment:
Nitpick: Looking at `WorkspaceState`, it's hard to tell that this model is scoped to SPM. While that might be obvious when working on the SPM's codebase, I think it'll lead to confusion in Tuist's domain. Therefore, I propose renaming it to something like `SwiftPackageManagerWorkspaceState` or something along those lines.